.github/workflows/kernel_test.yml

```diff
@@ -82,4 +82,6 @@ jobs:
         
       - name: Regression Test (Multiboot2)
         id: regression_test_linux
-        run: make run AUTO_TEST=regression ENABLE_KVM=0 BOOT_PROTOCOL=multiboot2 RELEASE=1
+        run: |
+          make clean
+          make run AUTO_TEST=regression ENABLE_KVM=0 BOOT_PROTOCOL=multiboot2 RELEASE=1
```

Comment:
`make clean` is the workaround, right? If so, remove it.

Reply:
Yes. Now we have another solution of changing `BOOT_PROTOCOL`. I will remove it.